### PR TITLE
REDAME: clear our section about docker storage

### PR DESCRIPTION
**Pull request by lukas-bednar**
Fixes: #132

Signed-off-by: Lukas Bednar <lbednar@redhat.com.com>

### CODE REVIEW

File: playbooks/README.md [hunk 1]
```diff
@@ -79,9 +86,11 @@ $ ansible-playbook -i inventory playbooks/cluster/kubernetes/config.yml
 ### OpenShift cluster
 
 
-Be sure that you have enough space on your machines for docker storage and
-modify [defaults values][docker-storage-setup-defaults] accordingly.
-Follow [docker-storage-setup] documentation for more details.
+Be sure that you have **extra disk** attached on your machines
```
Comment (alexxa) [inline playbooks/README.md]: AN extra disk
attached TO, not on

File: playbooks/README.md [hunk 2]
```diff
@@ -79,9 +86,11 @@ $ ansible-playbook -i inventory playbooks/cluster/kubernetes/config.yml
...
+for **docker storage** and modify [defaults values][docker-storage-setup-defaults]
+accordingly. In the most of cases you need to set `docker_dev` variable
```
Comment (alexxa) [inline playbooks/README.md]: No THE required here, simply: In most cases

File: playbooks/README.md (hunk 1 above)
Comment (lukas-bednar) [reply playbooks/README.md]: addressed

File: playbooks/README.md (hunk 2 above)
Comment (lukas-bednar) [reply playbooks/README.md]: addressed

File: playbooks/README.md [hunk 3]
```diff
@@ -27,6 +27,12 @@ For OpenShift deployment clone [**OpenShift Ansible**][openshift-ansible-project
 $ git clone -b release-3.7 https://github.com/openshift/openshift-ansible
 ```
 
+or for OpenShift 3.9
+
+```bash
+$ git clone -b openshift-ansible-3.9.0-0.40.0 https://github.com/openshift/openshift-ansible
```
[the remark this answers is not in the document]
Comment (lukas-bednar) [reply playbooks/README.md]: addressed

File: playbooks/README.md [hunk 4]
```diff
@@ -27,6 +27,10 @@ For OpenShift deployment clone [**OpenShift Ansible**](openshift-ansible-project
 $ git clone -b release-3.7 https://github.com/openshift/openshift-ansible
 ```
 
+**NOTE:** For OpenShift 3.9 there is no release yet,
+please read [this file](../automation/check-patch.openshift_3-9.sh) to see
```
[the remark this answers is not in the document]
Comment (lukas-bednar) [reply playbooks/README.md]: done

File: playbooks/README.md [hunk 5]
```diff
@@ -68,6 +72,7 @@ master.example.com
 ```
 
 Using this inventory file one can deploy a Kubernetes or OpenShift cluster.
+Make sure that you have **passwordless ssh access** to all members of cluster.
```
[the remark this answers is not in the document]
Comment (lukas-bednar) [reply playbooks/README.md]: done

File: playbooks/README.md (hunk 1 above)
Comment (lukas-bednar) [reply playbooks/README.md]: lost in rebase, updated

File: playbooks/README.md (hunk 2 above)
Comment (lukas-bednar) [reply playbooks/README.md]: lost in rebase, updated

**Comment by alexxa** [comment]
Thank you @lukas-bednar

**Pull request merged**